src/main/java/com/conveyal/r5/streets/Split.java

```diff
@@ -48,6 +53,8 @@ public void setFrom (Split other) {
         distSquared = other.distSquared;
```

Comment:
We should be able to find a fairly accurate distance with `VertexStore.fixedDegreesToFloating(FastMath.sqrt(distSquared)) * metersPerDegreeLat * 1000;` and such a line deserves a comment about what's going on.

Reply:
Addressed in https://github.com/conveyal/r5/pull/358/commits/440e107de872429dd5c0da2461cbcd7bd0a7d945